libfwupdplugin/fu-pefile-firmware.c

```diff
@@ -90,6 +92,7 @@ fu_pefile_firmware_parse_section(FuFirmware *firmware,
 		fu_csv_firmware_add_column_id(FU_CSV_FIRMWARE(img), "vendor_package_name");
 		fu_csv_firmware_add_column_id(FU_CSV_FIRMWARE(img), "$version");
 		fu_csv_firmware_add_column_id(FU_CSV_FIRMWARE(img), "vendor_url");
+		truncate_to_nul = TRUE;
```

Comment:
Are you sure this is needed? I have looked at the .sbat section in the file posted in #6055, and there was no null character. Unless we do not read the section size correctly?

Reply:
See #6091

Reply:
Hah, good catch. Thanks.